runtime/gc_modron_startup/mmparseXXgc.cpp

```diff
@@ -1530,6 +1530,16 @@ gcParseXXgcArguments(J9JavaVM *vm, char *optArg)
 			continue;
 		}
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+		if (try_scan(&scan_start, "RestoreGCThreads=")) {
```

Comment:
hm, I'd rather not introduce a new option, but rely on existing ones
it's just that the option parsing logic would be extended to know if it's a restore (vs fresh VM start) so that value is stored into special restoreGCthreadCount variable.

Reply:
you can look at `vm->checkpointState.restoreArgsList` with something like `FIND_AND_CONSUME_ARG` to search for restore options